neptune/new/exceptions.py

```diff
@@ -704,3 +704,23 @@ def __init__(self):
 """
         inputs = dict(list({}.items()) + list(STYLES.items()))
         super().__init__(message.format(**inputs))
+
+
+class NeptuneLegacyIncompatibilityException(NeptuneException):
+    def __init__(self):
+        message = """
+{h1}
+----NeptuneLegacyIncompatibilityException----------------------------------------
+{end}
+It seems you are passing the legacy Experiment object, when a Run object is expected.
+
+What can I do?
+    - Updating your code to the new Python API requires few changes, but to help you with this process we prepared a handy migration guide:
+    https://docs.neptune.ai/migration-guide
+    - You can read more about neptune.new in the release blog post:
+    https://neptune.ai/blog/neptune-new
+
+{correct}Need help?{end}-> https://docs.neptune.ai/getting-started/getting-help
+"""
+        inputs = dict(list({}.items()) + list(STYLES.items()))
+        super().__init__(message.format(**inputs))
```

Comment:
Maybe just `super().__init__(message.format(**STYLES))`?

Reply:
I think it's some kind of convention we have -- it's the same in all other exceptions, so I wanted to stay consistent

Reply:
I've already break it into other PR :wink:

Reply:
so un-break it, consistency is good :P

if you think this is a stupid convention (I won't really argue), we can change it in all instances, in a separate PR